src/extension/wps-download/src/main/java/org/geoserver/wps/gs/download/DownloadAnimationProcess.java

```diff
@@ -107,12 +108,26 @@ public RawData execute(
                     Integer headerHeight,
             @DescribeParameter(
                         name = "time",
-                        min = 1,
+                        min = 0,
                         description =
                                 "Map time specification (a range with "
                                         + "periodicity or a list of time values)"
                     )
                     String time,
+            @DescribeParameter(
+                        name = "animateParam",
```

Comment:
So these two are equivalents of "aParam", "aValues" of the old animator, right?

Reply:
see discussion on top, still need to decide if it shall handle all possible params or just env params